src/components/views/JobForm.js

```diff
@@ -104,7 +107,7 @@ class JobForm extends React.Component {
               <TitleManager record={create}>
                 <form>
                   <div className={css.jobForm}>
-                    {format === 'KBART' && <KbartFields /> }
+                    {format === 'KBART' && <KbartFields localKB={this.props.localKB} /> }
```

Comment:
When we're already destructuring props in a function like here on L94, it's preferable to keep it consistent. 

(Maybe put handlers, format, and localKB on separate lines now that we're destructuring three props and the line is getting long.)

Reply:
Yep, makes sense, tweaks made